src/OsmMom.sol

```diff
@@ -75,23 +71,39 @@ contract OsmMom {
 
     mapping (bytes32 => address) public osms;
 
-    constructor() public {
+    constructor(address vat_) {
+        vat = vat_;
         owner = msg.sender;
+        emit SetOwner(msg.sender);
+    }
+
+    function file(bytes32 what, address data) external onlyOwner {
+        if (what == "autoLine") {
+            autoLine = data;
+        } else {
+            revert("OsmMom/file-unrecognized-param");
```

Comment:
Previous error messages say `osm-mom` (not `OsmMom`).

Reply:
Changed the prev two to `OsmMom` so we standardize with what we do everywhere.